jwst/datamodels/multiexposure.py

```diff
@@ -62,7 +62,13 @@ def __init__(self, init=None, **kwargs):
             self.exposures[0].data = init.data
             self.exposures[0].dq = init.dq
             self.exposures[0].err = init.err
+            self.exposures[0].wavelength = init.wavelength
+            self.exposures[0].barshadow = init.barshadow
             self.exposures[0].area = init.area
+            self.exposures[0].var_poisson = init.var_poisson
+            self.exposures[0].var_rnoise = init.var_rnoise
+            self.exposures[0].var_flat = init.var_flat
+            self.exposures[0].pathloss = init.pathloss
```

Comment:
What happens if the input `MultiSlitModel` or `SlitDataModel` does not have one of these data arrays?  The new `MultiExposureModel` now will, but with the default values populated.

Reply:
The results of 1 test (so far) for inputs that do not have BARSHADOW or PATHLOSS arrays shows that they don't show up in the outputs. Only things that actually exist in the input get copied.

Reply:
Ah good!  Whew.